webapp/app/static/css/base.css

```diff
@@ -23,10 +23,10 @@
 }
 
 html {
-    font-size: 16px;
+    font-size: 17px;
```

Comment:
I have some questions about this, but if you've already considered these issues then please ignore my comment! I'm by no means an expert on font sizing in CSS (my knowledge is probably about as deep as this MDN article: https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Fundamentals#font_size)

Isn't 17px a difficult base size because almost all multiples we'd use are fractional?

E.g.:
- `--text-s` is defined as 0.9rem => 15.3px
- `--text-medium` is defined as 1.125rem => 19.125px
- `--text-xl` is defined as 1.375rem => 23.375px

With a base of 16px, most of these multiples worked out to round integer values: 1.125rem = 18px, 1.375 = 22px, etc.

I believe browsers all differ in how they render fractional pixels and this can cause aliasing issues and may make text look wonky.

I'm guessing that the designs provide rounded font sizes already? One way I know of getting these exactly as in the design is to set the base font size on `html` to 10px for easy math and then define the `--text-...` variables to be exactly what the designs provide, e.g. `--text-s: 1.5rem` (15px), `--text-base: 1.7rem` (17px), etc pp. (These are examples, obviously use whatever values are actually desired.)

cc @na-st